Automattic/pocket-casts-ios
# Developer: Database Export/Import file

* Adds file handling for a `pcasts` file
* Imports preferences and database from the file & kills the app
* Option to delete the DB & Settings file by just deleting and resetting `UserDefaults` instead of incrementally unsubscribing (as "Unsubscribe from all Podcasts" does)

https://github.com/Automattic/pocket-casts-ios/assets/3250/17e1d388-6287-452c-8814-88e2cc12bd20

## To test

#### Test Bundle
* Visit Developer Menu
* Tap "Export Bundle"
* Save in Files
* Tap the "Reset Database + Settings" - app will quit
* Visit Developer Menu
* Tap "Import Bundle" - app will quit
* Select the `pcasts` file
* Open the app and ensure podcasts and settings match

#### Test import from support export
* Go to Profile > Help & Feedback > ⋯ > Export Database
* Share the file to your computer
* Unzip the file
* Append the `pcasts` extension to the folder
* Delete and re-run the app
* Drag and drop this file onto the simulator
* Hit the "Import" option when prompted
* Podcasts and settings should match

## Checklist
